internal/backend/remote-state/s3/backend_test.go

```diff
@@ -1593,7 +1593,7 @@ func testGetWorkspaceForKey(b *Backend, key string, expected string) error {
 }
 
 func checkStateList(b backend.Backend, expected []string) error {
-	states, err := b.Workspaces()
+	states, err := b.Workspaces(context.TODO())
```

Comment:
Wouldn't be better to get `context.Context` as argument and use `t.Context()` for that?